src/game/Board.java

```diff
@@ -0,0 +1,48 @@
+package game;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Board {
+	private final Map<Integer,Position> map;
```

Comment:
why map, not array/list

Reply:
also variable name is not self explanatory